frontend/src/store/config.js

```diff
@@ -230,19 +236,19 @@ export const useConfigStore = defineStore('config', () => {
   })
 
   const controlPlaneHighAvailabilityHelp = computed(() => {
-    return state.value?.shootDefaults?.controlPlaneHighAvailabilityHelp ?? state.value?.controlPlaneHighAvailabilityHelp
+    return state.value?.shootDefaults.controlPlaneHighAvailabilityHelp ?? state.value?.controlPlaneHighAvailabilityHelp
```

Comment:
```suggestion
    return shootDefaults.controlPlaneHighAvailabilityHelp ?? state.value?.controlPlaneHighAvailabilityHelp
```

Add a computed for shootDefaults. See also my comment. Adapt other computed properties accordingly